src/window.rs

```diff
@@ -952,6 +952,16 @@ impl Window {
     pub fn theme(&self) -> Option<Theme> {
         self.window.theme()
     }
+
+    /// Gets the current title of the window.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **iOS / Android / x11 / Wayland / Web:** Unsupported.
```

Comment:
should add that we return an empty string in this case.

Reply:
Sure